src/main/resources/templates/db_scripts/setEffectiveHoldingsLocation.sql

```diff
@@ -1,11 +1,4 @@
-START TRANSACTION;
-
 UPDATE ${myuniversity}_${mymodule}.holdings_record
-  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}', TO_JSONB(holdings_record.jsonb->>'temporaryLocationId'))
-WHERE jsonb->>'temporaryLocationId' IS NOT NULL;
-
-UPDATE ${myuniversity}_${mymodule}.holdings_record
-  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}', TO_JSONB(holdings_record.jsonb->>'permanentLocationId'))
-WHERE jsonb->>'temporaryLocationId' IS NULL;
-
-END TRANSACTION;
\ No newline at end of file
+  SET jsonb = JSONB_SET(holdings_record.jsonb, '{effectiveLocationId}',
+                        TO_JSONB(COALESCE(temporarylocationid, permanentlocationid)))
+WHERE effectivelocationid IS NULL;
```

Comment:
Where does the value of this come from?

There does not seem to be any assignment (the case is different to the case of the only other usage).

Reply:
https://github.com/folio-org/raml-module-builder#foreign-keys-constraint

For each foreign key RMB creates a trigger that automatically copies the foreign key value from the jsonb field into a dedicated field. This is needed to create the database index and to create the foreign key relationship constraint.

This applies to the three fields temporarylocationid, permanentlocationid, effectivelocationid.

Reply:
Thank you, that was context I was missing.